tests/pr_test.go

```diff
@@ -14,9 +14,10 @@ const resourceGroup = "geretain-test-base-ocp-vpc"
 const standardExampleTerraformDir = "examples/standard"
 
 // Ensure there is one test per supported OCP version
-const ocpVersion1 = "4.10"
-const ocpVersion2 = "4.9"
-const ocpVersion3 = "4.8"
+const ocpVersion1 = "4.12"
```

Comment:
I would suggest updating `setupOptions` to use latest, and ensure each of the other tests cover the others

Reply:
Noted, will change this. Other versions are covered as part of `other_tests.go`